packages/ppx/test/bucklescript/src/__snapshots__/array_test.bs.js.snap

```diff
@@ -41,6 +42,7 @@ exports[`SequenceDynamicComponent renders 1`] = `
 <div>
   <div
     class="emotion-0"
+    var="White"
```

Comment:
Does that seem like an accident? We don't want to render variables into dom attributes (I fixed this last month). Don't worry about it, I will handle it. Try to open an issue if you merge this PR

Edit: This is the cause of failing the CI.